platforms.txt

```diff
@@ -0,0 +1 @@
+linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x
```

Comment:
It doesn't really make sense to build for more platforms than [we build Icinga 2 containers for](https://github.com/Icinga/docker-icinga2/blob/master/platforms.txt).